.travis.yml

```diff
@@ -23,5 +23,13 @@ before_script:
 script:
   - py.test
   - npm test
+  - pip uninstall -y ipywidgettestwidgets
+  - pip install .
+  - jupyter nbextension enable --py --sys-prefix ipywidgettestwidgets
+  - pip install jupyterlab
+  - jupyter labextension list
+  - jupyter labextension list 2>&1 | grep -q jupyter-widget-testwidgets
+  - jupyter labextension install @jupyter-widgets/jupyterlab-manager --no-build
+  - jupyter labextension link
```

Comment:
These commands seems a bit like black magic. What do they do? Would you mind adding a comment line?